src/persist/src/operators/stream.rs

```diff
@@ -303,6 +308,57 @@ where
     }
 }
 
+/// Extension trait for [`Stream`].
+pub trait AwaitFrontier<G: Scope<Timestamp = u64>, D> {
+    /// Stashes data until it is no longer beyond the input frontier.
+    ///
+    /// This is similar, in spirit, to what `consolidate()` does for differential collections and
+    /// what `delay()` does for timely streams. However, we are not working with differential
```

Comment:
> However, we are not working with differential collections
We're not?

Concretely, I think I mean to say that this is misleading. We could easily make these Streams into Collections, they have both the required type and (more importantly) the matching semantics. I think the reason we don't want consolidate is that it does extra work that we don't need

Reply:
True, I'll change that to say that we don't want it because of the extra work.